server.js

```diff
@@ -6,10 +6,11 @@ var verifier = require("google-id-token-verifier");
 var fs = require("fs");
 var mkdirp = require("mkdirp");
 
+var serverIpAddress = process.env.SERVER_IP_ADDRESS || "127.0.0.1";
 var port = process.env.PORT || 8080;
 var oauthClientId = "627385202945-oqedl0onib41h39quc15pufqgqp8j8cu.apps.googleusercontent.com";
```

Comment:
Maybe this pull request is a good place to make this an environment variable so we can use separate production and dev oauth credentials. Potential for faf though so your call.